.vscode/settings.json

```diff
@@ -1,7 +1,20 @@
 {
-    "python.testing.pytestArgs": [
-        "."
+    "python.linting.pylintEnabled": true,
+    "python.linting.flake8Enabled": true,
+    "python.linting.mypyEnabled": true,
+    "python.linting.enabled": true,
+    "python.formatting.provider": "black",
+    "editor.formatOnSave": true,
+    "editor.rulers": [
+        88
     ],
     "python.testing.unittestEnabled": false,
```

Comment:
Do we want this to be 'true'?

Reply:
I don't think so.

Basically there are multiple unit testing frameworks available in Python and I *think* you guys are using ``pytest``, rather than the built-in ``unittest`` one.

@davidorme Have I got that right?

Reply:
Got it! So I am resolving this conversation.

Reply:
> The testing options are for VSCode. Just `setup.cfg` or `pyproject.toml` is not enough.

So I *think* we can drop the ``pytest`` arguments ``-s tests`` because if the test directory is set appropriately in e.g. ``setup.cfg`` then I think VS code will detect it (and so the test tab will work properly) and if it's only set in ``settings.json`` then VS code's test tab will work, but confusingly, it won't work when you run ``pytest`` from the command line (or when the CI system runs it).